pkg/api/types.go

```diff
@@ -2450,3 +2450,70 @@ const (
 	// "default-scheduler" is the name of default scheduler.
 	DefaultSchedulerName = "default-scheduler"
 )
+
+// Address of a cluster
```

Comment:
Actually according to the doc [here](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api_changes.md#so-you-want-to-change-the-api) and [here](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api_changes.md#incompatible-api-changes), we should add new API to the extensions group first, and then graduate it to its own group.